workbench-core/example/app/src/routes/auditRoutes.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuditService, Metadata, MetadataParser } from '@aws/workbench-core-audit';
+import { isAuditIncompleteError } from '@aws/workbench-core-audit/lib/errors/auditIncompleteError';
+import { DynamicAuthorizationService } from '@aws/workbench-core-authorization';
+import { validateAndParse } from '@aws/workbench-core-base';
+import * as Boom from '@hapi/boom';
+import { Request, Response, Router } from 'express';
+import { wrapAuth } from '../utilities/authWrapper';
+
+export function setupAuditRoutes(
+  router: Router,
+  auditService: AuditService,
+  dynamicAuthorizationService: DynamicAuthorizationService
+): void {
+  router.post(
+    '/audit/write',
```

Comment:
probably don't need `write` in this path

Reply:
fixed